presto-geospatial/src/test/java/io/prestosql/plugin/geospatial/TestSpatialJoins.java

```diff
@@ -13,9 +13,10 @@
  */
```

Comment:
`- rename HdfsConfigurationUpdater to HdfsConfigurationInitializer` should be in separate commit (or dropped)

Reply:
Yes, and it was before I suggested changing the name (probably lost in the rebase update)